packages/paste-core/components/sidebar/src/header/SidebarHeaderIconButton.tsx

```diff
@@ -2,14 +2,19 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import {Button} from '@twilio-paste/button';
 import type {ButtonProps} from '@twilio-paste/button';
+import type {BoxProps} from '@twilio-paste/box';
 
 export interface SidebarHeaderIconButtonProps extends Omit<ButtonProps, 'size' | 'variant'> {
   children: NonNullable<React.ReactNode>;
+  as: 'a' | 'button';
+  onClick?: ButtonProps['onClick'];
+  href?: ButtonProps['href'];
+  element?: BoxProps['element'];
 }
 
 export const SidebarHeaderIconButton = React.forwardRef<HTMLButtonElement, SidebarHeaderIconButtonProps>(
   ({element = 'SIDEBAR_HEADER_ICON_BUTTON', ...props}, ref) => {
-    return <Button {...props} size="icon" variant="inverse" element={element} ref={ref} />;
+    return <Button {...props} tabIndex={0} size="icon" variant="inverse" element={element} ref={ref} />;
```

Comment:
question: we don't necessarily need tabIndex=0 here, do we? or is this 200iq for JS users

Reply:
I think this was just a weird merge conflict I got when resolving pulling in main. Shouldn't be there